curriculum/challenges/english/01-responsive-web-design/css-grid/use-grid-column-to-control-spacing.english.md

```diff
@@ -77,16 +79,60 @@ tests:
 
 </div>
 
+### After Test
+<div id='html-setup'>
 
+```html
+<script>
+const hasCorrectSpacing = () => {
+  const contTwoPlusThreePlusGapWidth = $('.item2').width() * 2 + 10;
+  const item5Width = $('.item5').width();
+  const diff = Math.abs(contTwoPlusThreePlusGapWidth - item5Width);
+  /* max expected diff assumes browser width increments of 0.01px */
```

Comment:
```suggestion
  /* To avoid rounding errors the largest allowed diff is set at 0.01px */
```
I found the original wording a little confusing. That said, precisely what browsers do with widths is a little confusing...